thorvg/examples/thorvg-example/sdkconfig.bsp.esp32_p4_function_ev_board

```diff
@@ -0,0 +1,17 @@
+# This file was generated using idf.py save-defconfig. It can be edited manually.
+# Espressif IoT Development Framework (ESP-IDF) Project Minimal Configuration
+#
+CONFIG_IDF_TARGET="esp32p4"
+
```

Comment:
I wonder if we can rename the `sdkconfig.bsp.esp32_p4_function_ev_board` to `sdkconfig.defaults.esp32p4` because the default configs list here are not specific the **p4 function ev board**, other esp32p4 board can also use it.